### ISSUE TITLE
CUDA CustomHBondForce with more than four exclusions per donor?

### ISSUE THREAD

**Issue by mvanvleet** [pandegroup/openmm#1787]
I'm looking to implement a force field for chloromethane (CClH3) that sets a CustomHbondForce on each of the atoms. Unfortunately, my current force field generates five exclusions per donor, which is apparently [not allowed in the CUDA platform.](https://github.com/pandegroup/openmm/blob/master/platforms/cuda/src/CudaKernels.cpp#L4379) Due to simulation speed, the CUDA platform is much preferable to the CPU platform.

If need be, I will find a workaround for this issue. However, if it would be trivial to enable CUDA simulations where CustomHBondForce has more than four exclusions per donor, that would be incredibly useful to me.

**Comment by mvanvleet** [comment]
@peastman , is this a trivial fix, or should I find a workaround?

**Comment by peastman** [comment]
This would be complicated to change, so I suggest looking for a workaround.

**Comment by jchodera** [comment]
Does OpenCL support more than five exclusions/donor?

**Comment by peastman** [comment]
No, the implementations for CUDA and OpenCL are almost identical.
